addOns/requester/src/test/java/org/zaproxy/addon/requester/internal/HttpPanelSenderUnitTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2023 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.requester.internal;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+
+import java.io.IOException;
+import org.apache.commons.httpclient.URI;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.parosproxy.paros.network.HttpHeaderField;
+import org.parosproxy.paros.network.HttpMessage;
+import org.parosproxy.paros.network.HttpRequestHeader;
+
+class HttpPanelSenderUnitTest {
+
+    private static final String URI = "http://example.com";
+    private HttpMessage msg;
+
+    @BeforeEach
+    void init() throws IOException {
+        HttpRequestHeader requestHeader = new HttpRequestHeader();
+        requestHeader.setURI(new URI(URI, false));
+
+        msg = new HttpMessage();
+        msg.setRequestHeader(requestHeader);
+    }
+
+    @Test
+    void shouldLowercaseheaderNames() {
```

Comment:
`…CaseHeader…`

Reply:
This and following are pending.

Reply:
Sorry for that!, Now its Done.

Reply:
No worries, there's also https://github.com/zaproxy/zap-extensions/pull/5149#discussion_r1424328009